willywax/NextProduct
# #167974864 set-up sequelize and create database models

#### Description of Task to be completed?
Set up sequelize and create the database models(structure)
#### How should this be manually tested?
- Clone this repository
- Checkout this branch.  run `npm install` to install all dependencies. 
- Make sure you have Postgres installed and is up running.
- Create your development database
- Set-up .env file inline with the .env.example file
- Export your database connection string to the terminal 
`export DATABASE_URL=postgres://username:password@localhost:5432/databasename`
- Run `sequelize db:migrate` if you have sequelize-cli installed globally or
`node_modules/.bin/sequelize db:migrate`
- See if the tables are created in the database.
#### What are the relevant pivotal tracker stories?
[#167974864](https://www.pivotaltracker.com/story/show/167974864)
